config/vufind/ProQuestFSG.ini

```diff
@@ -0,0 +1,160 @@
+; This section contains global settings affecting search behavior.
+; For ProQuest Federated Search Gateway API documentation:
+; https://support.proquest.com/s/article/ProQuest-Academic-Platform-Federated-Search-Options
+[General]
+; Enable the ProQuest Federated Search Gateway integration.  Disabled by default.
+;enabled = true
+
+; This setting controls the default sort order of search results; the selected
+; option should be one of the options present in the [Sorting] section below.
+default_sort         = relevance
+; This section controls the result limit options for search results. default_limit
+; sets the default number of results per page. limit_options is a comma-separated
+; list of numbers to be presented to the end-user. If only one limit is required,
+; set default_limit and leave limit_options commented out.
+; WARNING: using large limits may cause problems due to a variety of limitations,
+; especially if you support bulk operations (which can cause large URLs/requests).
+; If you must support large page sizes, you may need to raise the PHP memory_limit
+; and max_input_vars settings and/or adjust the Apache LimitRequestLine setting.
+default_limit        = 20
+; limit_options        = 10,20,40,60,80,100
+
+; This setting allows to limit pagination of a search result as deep pagination
+; costs a lot of performance and most users are not very likely to navigate
+; further down than 20 pages of a search result.
+; This is especially useful to prevent webcrawlers from descending too deep and
+; eating up search backend performance. 
+; Default is 400; the API supports a maximum limit of 1000.
+;result_limit = 400
+
+; These are the default recommendations modules to use when no specific setting
+; are found in the [TopRecommendations], [SideRecommendations] or
+; [NoResultsRecommendations] sections below.
+; See the comments above those sections for details on legal settings.  You may
+; repeat these lines to load multiple recommendations.
+;default_top_recommend[] = TopFacets:FacetsTop:Primo
```

Comment:
Stray Primo reference here. Should we just remove this line?

Reply:
This one is not necessarily wrong (you could??? bring in Primo results at the top) but it's definitely confusing and shout not be there.  Removed.